### PR TITLE
[Portal] Docs: Winston/cnct 2314 consolidate documentation for in app wallets and ecosystem

### PR DESCRIPTION
**Pull request by ElasticBottle**
https://linear.app/thirdweb/issue/CNCT-2314/consolidate-documentation-for-in-app-wallets-and-ecosystem

---

## PR-Codex overview
This PR focuses on the restructuring and enhancement of the `thirdweb` wallet documentation, including the removal of outdated files, the addition of new features, and the reorganization of content to improve clarity and accessibility for users.

### Detailed summary
- Deleted multiple outdated `.mdx` files related to `sign-in` and `ecosystems`.
- Introduced new `AuthList` component for displaying authentication options.
- Updated wallet security documentation to include enclave security details.
- Improved organization of wallet-related guides and resources.
- Added new content on guest mode, linking identities, and pregenerating wallets.
- Enhanced the sidebar for better navigation within wallet documentation.
- Revised existing pages to reflect new wallet features and configurations.

> The following files were skipped due to too many changes: `apps/portal/src/app/connect/sidebar.tsx`, `apps/portal/src/app/connect/wallet/faq/page.mdx`, `apps/portal/src/app/connect/wallet/user-management/get-user-profiles/page.mdx`, `apps/portal/redirects.mjs`, `apps/portal/src/app/connect/wallet/security/assets/ecosystem-scoped.svg`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/portal/src/app/connect/sidebar.tsx [hunk 1]
```diff
@@ -90,126 +91,182 @@ export const sidebar: SideBar = {
...
+                // },
+              ],
+            },
+            {
+              name: "Ecosystem",
+              links: [
                 {
-                  name: "Compact Modal",
-                  href: `${connectSlug}/customization#compact-modal`,
+                  name: "Ecosystem Wallet Explorer Page",
```
Comment (gregfromstl) [inline apps/portal/src/app/connect/sidebar.tsx:214]: We're officially calling this the "Ecosystem Portal"

File: apps/portal/src/app/connect/sidebar.tsx [hunk 2]
```diff
@@ -90,126 +91,182 @@ export const sidebar: SideBar = {
       name: "Learn",
       isCollapsible: false,
       links: [
-        // Connect
+        // Connect\
         {
-          name: "Sign-In",
-          icon: <WalletsConnectIcon />,
+          name: "Wallets",
+          icon: <WalletsInAppIcon />,
```
Comment (gregfromstl) [inline apps/portal/src/app/connect/sidebar.tsx:97]: Same icon as In-App Wallets?

File: apps/portal/src/app/connect/wallet/overview/page.mdx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+	<FeatureCard
+		title="User Attribution & Analytics"
+		description="Unlock analytics for developers and end user activity across your ecosystem"
+		iconUrl="/icons/feature-cards/compatibility.svg"
+	/>
+</div>
+
+### Solutions
+
+| SOLUTION | BEST FOR | PRICING |
```
Comment (gregfromstl) [inline apps/portal/src/app/connect/wallet/overview/page.mdx:76]: Love this

File: apps/portal/src/app/connect/wallet/overview/page.mdx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+|----------|----------|----------|
+| Connect | Applications needing traditional EOA wallet integration | Free |
+| In-App Wallet | Mainstream applications requiring seamless onboarding | [Free up to 1,000 MAW, then $0.02/MAW](https://thirdweb.com/pricing) |
+| Ecosystem Wallet | Organizations building branded wallet networks | [From $250/mo per instance](https://thirdweb.com/pricing) |
```
Comment (gregfromstl) [inline apps/portal/src/app/connect/wallet/overview/page.mdx:80]: Can probably just say $250/mo here, no "From" or "per instance" needed

File: apps/portal/src/app/connect/wallet/overview/page.mdx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+		iconUrl="/icons/feature-cards/compatibility.svg"
+	/>
+</div>
+
+### Solutions
+
+| SOLUTION | BEST FOR | PRICING |
+|----------|----------|----------|
+| Connect | Applications needing traditional EOA wallet integration | Free |
+| In-App Wallet | Mainstream applications requiring seamless onboarding | [Free up to 1,000 MAW, then $0.02/MAW](https://thirdweb.com/pricing) |
```
Comment (gregfromstl) [inline apps/portal/src/app/connect/wallet/overview/page.mdx:79]: Only 1,000? I thought it was a lot more than that? Also I think it's just wallets not monthly active, right?

File: apps/portal/src/app/connect/wallet/sign-in-methods/external-wallets/page.mdx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# External wallets
+
+thirdweb supports a number of first-party web3 wallets such as Metamask and Phantom.
```
Comment (gregfromstl) [inline apps/portal/src/app/connect/wallet/sign-in-methods/external-wallets/page.mdx:3]: "thirdweb natively supports 350+ first-party wallets and all wallets that implement EIP-6963 (Multi-injected Provider Discovery)"

**Review by joaquim-verges — APPROVED** [review]

**Comment by joaquim-verges** [comment]
### Merge activity

* **Nov 19, 5:30 PM EST**: The merge label 'merge-queue' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=thirdweb-dev&repo=js) once it meets the requirements.

**Pull request merged**